conbench/templates/app.html

```diff
@@ -86,7 +86,7 @@ <h4 class="modal-title"><i class="bi bi-trash3-fill"></i></h4>
             <!-- Categories: success (green), info (blue), warning (yellow), danger (red) -->
             {% for category, message in messages %}
             <div class="alert alert-{{ category }}" role="alert">
-                <i class="bi bi-lightning"></i> {{ message }}
+                <p class="fs-5"><i class="bi bi-exclamation-diamond-fill"></i> {{ message }}</p>
```

Comment:
two changes:
- this icon: https://icons.getbootstrap.com/icons/exclamation-diamond-fill/
- this font size: https://getbootstrap.com/docs/5.2/utilities/text/#font-size